src/components/DigitalAssetsGrid/AddAsset/AddAsset.js

```diff
@@ -0,0 +1,24 @@
+// @flow
```

Comment:
Why renamed file looks like new file in git?

Reply:
it was not only renamed